infra/onert-micro/CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
When I try to build with this version of cmake, I got:
```
CMake Error at /home/aefimov/nnfw/infra/nncc/cmake/buildtool/config/arm-none-eabi-gcc.cmake:16 (list):
  list does not recognize sub-command POP_FRONT
```

POP_FRONT is available from 3.15 ([see](https://cmake.org/cmake/help/latest/command/list.html#pop-front))